### ISSUE TITLE
Make PackageName default to ChocolateyPackageName

### ISSUE THREAD

**Issue by FranklinYu** [chocolatey/choco#1716]
Following helpers have a parameter `-PackageName`:
- `Install-ChocolateyZipPackage`
- `Get-ChocolateyUnzip`

When they are used in install scripts (which is likely to be the case), this parameter seems redundant since we already have `$env:ChocolateyPackageName`. It would be great to let it default to `$env:ChocolateyPackageName` when this environment variable is available.

**Comment by justinwalker4179** [comment]
I am a new contributor to this project and was wondering if I could be assigned this issue.

**Comment by cowmanjoe** [comment]
I'm going to look into this as a first issue.

**Comment by cowmanjoe** [comment]
Would this change not also apply to `Install-ChocolateyPackage` as well?

**Comment by FranklinYu** [comment]
> Would this change not also apply to `Install-ChocolateyPackage` as well?

Yes I think so! Thank you for your Pull Request.

**Comment by cowmanjoe** [comment]
Added this default value for several other scripts that use `$packageName`. If this is an optional parameter now that it has a default variable, shouldn't it not be in position 0? The problem is if we change the position, it breaks the API for other calling scripts.

**Comment by FranklinYu** [comment]
> The problem is if we change the position, it breaks the API for other calling scripts.

Yes, that would be a breaking change. Although it sounds nice to have optional parameters after all the required ones, it’s not worth it to break packages, especially corporate internal ones (we are able to update the community feed ourselves).

Note that we already have some of such “wrong parameters”, like [`-Url` parameter of `Install-ChocolateyZipPackage`](https://chocolatey.org/docs/helpers-install-chocolatey-zip-package#url-string), for the same reason I guess.
